models/Form.php

```diff
@@ -0,0 +1,259 @@
+       <html lang="en" style="font-family: sans-serif;">
+            <head>
+                <meta charset="UTF-8">
+            </head>
+            <body>
+            
+                <br/>Voici les informations que vous m\'avez envoyées :
+                <br/>
+                <br/>Votre nom utilisateur : '.$name.'
+                <br/>Adresse mail utilisée : '.$email.'
+                <br/>Sujet du message : '.$subject.'
+                <br/>Message en entier : '.$message.'
+                <br/>
+                <br/>Je vous remercie pour votre message et vous répondrai dans un délai de 5 jours.
+                <br/>
+                <br/>Excellente journée !
+                <br/>
+                <br/>Jérôme L.
+                    
+
+            </body>
+        </html>
+    ';
+
+    $header = "MIME-Version: 1.0\r\n";
+    $header .= "Content-type: text/html; charset=UTF-8\r\n";
+    $header .= 'From: jerome.lacquemant@gmail.com' . "\r\n" . 'Reply-To: jerome.lacquemant@gmail.com' . "\r\n" . 'X-Mailer: PHP/' . phpversion();
+
+    mail($email,$subject_mail,$message_mail,$header);
+    }
+    
+ function form_page_postback(){
+        if(isset($_POST['delete'])){
+            $article = $model_article->delete_article();
+            $comment = $model_comment->delete_article_comments();
+        }
+
+        if(isset($_POST['submit'])){
+            if(isset($_POST['title'])){
+                $title = htmlspecialchars(trim($_POST['title']));
+            }
+            if(isset($_POST['content'])){
+                $content = htmlspecialchars(trim($_POST['content']));
+            }
+            $posted = isset($_POST['public']) ? "1" : "0";
+            
+            $errors = [];
+
+            if(empty($title) || empty($content)){
+                $errors['empty'] = "Veuillez remplir tous les champs svp";
+            }
+
+            if(!empty($_FILES['image']['name'])){
+                $file = $_FILES['image']['name'];
+                $extensions = ['.png','.jpg','.jpeg','.gif','.PNG','.JPG','.JPEG','.GIF'];
+                $extension = strrchr($file,'.');
+  
+                if(!in_array($extension,$extensions)){
+                    $errors['image'] = "Cette image n'est pas valable.";
+                }
+            }
+            
+            if(!empty($errors)){
+                ?>
+                <div class="card red">
+                    <div class="card-content white-text">
+                        <?php
+                        foreach($errors as $error){
+                            echo $error;
+                        }
+                        ?>
+                    </div>
+                </div>
+                <?php
+            }else{
+                $model_article->edit($title,$content,$posted,$_GET['id']);
+             
+                if(!empty($_FILES['image']['name']))
+                {
+                    $model_article->update_img($_FILES['image']['tmp_name'], $extension);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Direct use of $_FILES Superglobal detected.](https://app.codacy.com/manual/JeromeLacquemant/projet-5/pullRequest?prid=5344015)